.env.local

```diff
@@ -23,7 +23,13 @@ NX_SKIP_NX_CACHE=false
 # If set to `true`, will print debug information useful for troubleshooting.
 NX_VERBOSE_LOGGING=false
 
-### `nx-plugins` variables ###
+### Plugins variables ###
 
-# E2E package manager; `bun`, `npm`, `pnpm`, `yarn`, `infer`
-E2E_PACKAGE_MANAGER=
+# If set to `true`, will print E2E debug information useful for troubleshooting.
+CDWR_DEBUG_LOGGING=true
+
+# Package manager to use in E2E tests; `npm`, `pnpm`, `yarn` (todo: `bun`)
+CDWR_E2E_PACKAGE_MANAGER=
```

Comment:
Not set?

Reply:
No, it will then infer package manager from workspace, hence `yarn`